content/docs/gto/command-reference/assign.md

```diff
@@ -40,7 +40,9 @@ GTO creates a special Git tag in
 - `--simple <text>` - Use simple notation, e.g. `rf#prod` instead of `rf#prod-5`
   [supported values: auto, true, false] [default: auto]
 - `--force` - Create the Git tag even if it already exists and is in effect
-- `--push` - Push created tag automatically (experimental)
+- `--push` - Push created git tag to `origin`. The value is if the `repo` option
```

Comment:
```suggestion
- `--push` - Push created git tag to `origin`. Useful if the `repo` option
```

Minor, but I couldn't understand what this meant at first. I thought it meant to set the value of `--push` option to a remote url like `--push git@...`.

Reply:
good catch, thanks Dave. I meant actually that this option is _ignored_ if we are doing updates on the remote repo (we kinda have to push or create a tag remotely anyways). Update the text and merging, but pls let me know if I can improve anything else here.